dot/parachain/types/statement.go

```diff
@@ -80,33 +84,60 @@ type Seconded CommittedCandidateReceipt
 // Valid represents a statement that a validator has deemed a candidate valid.
 type Valid CandidateHash
 
-// statementVDTAndSigningContext is just a wrapper struct to hold both the statement and the signing context.
-type statementVDTAndSigningContext struct {
-	Statement StatementVDT
-	Context   SigningContext
+// encodeSignData encodes the statement and signing context into a byte slice.
+func encodeSignData(statement StatementVDT, signingContext SigningContext) ([]byte, error) {
```

Comment:
The name is a bit ambiguous and might suggest that the method also does the signing. Maybe `encodeWithSigningContext`?